### PR TITLE
added support for the INFO Severity and refactored the messages storage

### PR DESCRIPTION
**Pull request by Mason-Chou**
This is needed for Issue #1163. Currently the ValidationSupport model does not recognize messages that have the INFO Severity. This merge will add in that support.

Also refactor how messages are stored. Use one ArrayList instead of creating an Arraylist for each Severity type.

### CODE REVIEW

**Comment by Maxoudela** [comment]
Thanks for the PR.

I do not understand where the info message are stored? It seems that the `getMessages(null)` is only returning `invalidMessages` so any INFO message added will not be returned?

**Comment by Mason-Chou** [comment]
Before the changes, we have a multiple ArrayLists storing the **WARN** and **ERROR** ValidationMessage separately. I've consolidated this into one ArrayList and named it ```invalidMessage``` as I deemed it wasn't scalable.

I've changed the method ```add(ValidationMessage message)``` to add all ValdationMessage that does not have a ***null*** severity nor a ***OK*** severity.

Maybe the variable ```invalidMessages``` is confusing as I deemed any ValidationMessage without an **OK** severity to be invalid. If you have a better variable name, I would appreciate that suggestion.

**Comment by Maxoudela** [comment]
Hum yes thanks, I did not understand at first look, my bad. The Info message are indeed recorded into the `invalidMessages`. The variable name seems ok for me as OK messages are not recorded in it.

File: controlsfx/src/main/java/org/controlsfx/validation/ValidationResult.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2014, ControlsFX
+ * Copyright (c) 2014, 2019, ControlsFX
```
Comment (Maxoudela) [inline controlsfx/src/main/java/org/controlsfx/validation/ValidationResult.java]: Can you remove the last comma? As explained [here](https://github.com/controlsfx/controlsfx/wiki/Contributing-to-ControlsFX), the general format is :

> If you are editing an existing source file, the format should be , . For example, if the file was created in 2012, and it is currently 2014, it should say '2012, 2014'.

The rest seems good 👍

**Pull request merged**